{
  "stats": {
    "suites": 1,
    "tests": 3,
    "passes": 0,
    "pending": 0,
    "failures": 1,
    "start": "2023-03-07T09:51:20.353Z",
    "end": "2023-03-07T09:51:21.156Z",
    "duration": 781,
    "testsRegistered": 3,
    "passPercent": 0,
    "pendingPercent": 0,
    "other": 0,
    "hasOther": false,
    "skipped": 2,
    "hasSkipped": true
  },
  "results": [
    {
      "uuid": "f1c64572-572e-4918-8fa5-f28536f4d7e8",
      "title": "",
      "fullFile": "cypress\\integration\\promiseBasic.spec.js",
      "file": "cypress\\integration\\promiseBasic.spec.js",
      "beforeHooks": [],
      "afterHooks": [],
      "tests": [],
      "suites": [
        {
          "uuid": "2cfc728a-2636-43a2-9f42-5c1d5ae82d4d",
          "title": "Promise basic app",
          "fullFile": "",
          "file": "",
          "beforeHooks": [],
          "afterHooks": [],
          "tests": [
            {
              "title": "should resolve first promise by click on logo",
              "fullTitle": "Promise basic app should resolve first promise by click on logo",
              "timedOut": null,
              "duration": 781,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.get('.logo').click({\n  force: true\n});\ncy.get('.message').contains('Promise was resolved!');",
              "err": {
                "message": "TypeError: The following error originated from your application code, not from Cypress.\n\n  > promise1 is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n\nBecause this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Promise basic app`",
                "estack": "TypeError: The following error originated from your application code, not from Cypress.\n\n  > promise1 is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n\nBecause this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Promise basic app`\n    at Object.parcelRequire.scripts/main.js (http://localhost:8080/main.d8ebb8d6.js:141:1)\n    at newRequire (http://localhost:8080/main.d8ebb8d6.js:47:24)\n    at http://localhost:8080/main.d8ebb8d6.js:81:7\n    at http://localhost:8080/main.d8ebb8d6.js:120:3",
                "diff": null
              },
              "uuid": "9ecf845b-253b-4715-80f3-dcc4868e71d7",
              "parentUUID": "2cfc728a-2636-43a2-9f42-5c1d5ae82d4d",
              "isHook": false,
              "skipped": false
            },
            {
              "title": "should reject second promise in 3 seconds",
              "fullTitle": "Promise basic app should reject second promise in 3 seconds",
              "timedOut": null,
              "duration": 0,
              "state": "skipped",
              "speed": null,
              "pass": false,
              "fail": false,
              "pending": false,
              "context": null,
              "code": "cy.get('.error-message', {\n  timeout: 3000\n}).contains('Promise was rejected!');",
              "err": {},
              "uuid": "909735d2-10a5-4f93-b222-53ebdda8d61b",
              "parentUUID": "2cfc728a-2636-43a2-9f42-5c1d5ae82d4d",
              "isHook": false,
              "skipped": true
            },
            {
              "title": "shouldn't resolve any promise by right click",
              "fullTitle": "Promise basic app shouldn't resolve any promise by right click",
              "timedOut": null,
              "duration": 0,
              "state": "skipped",
              "speed": null,
              "pass": false,
              "fail": false,
              "pending": false,
              "context": null,
              "code": "cy.get('.logo').rightclick({\n  force: true\n});\ncy.get('.message').should('not.visible');",
              "err": {},
              "uuid": "3f0224e3-e118-4abb-bc0c-e003d55136fe",
              "parentUUID": "2cfc728a-2636-43a2-9f42-5c1d5ae82d4d",
              "isHook": false,
              "skipped": true
            }
          ],
          "suites": [],
          "passes": [],
          "failures": [
            "9ecf845b-253b-4715-80f3-dcc4868e71d7"
          ],
          "pending": [],
          "skipped": [
            "909735d2-10a5-4f93-b222-53ebdda8d61b",
            "3f0224e3-e118-4abb-bc0c-e003d55136fe"
          ],
          "duration": 781,
          "root": false,
          "rootEmpty": false,
          "_timeout": 2000
        }
      ],
      "passes": [],
      "failures": [],
      "pending": [],
      "skipped": [],
      "duration": 0,
      "root": true,
      "rootEmpty": true,
      "_timeout": 2000
    }
  ],
  "meta": {
    "mocha": {
      "version": "7.0.1"
    },
    "mochawesome": {
      "options": {
        "quiet": false,
        "reportFilename": "mochawesome",
        "saveHtml": false,
        "saveJson": true,
        "consoleReporter": "spec",
        "useInlineDiffs": false,
        "code": true
      },
      "version": "6.2.1"
    },
    "marge": {
      "options": {
        "reportDir": "raw_reports",
        "overwrite": false,
        "html": false,
        "json": true
      },
      "version": "5.1.0"
    }
  }
}